### PR TITLE
feat: add ability to have first line underneath the heading

### PR DESCRIPTION
**Pull request by lloydstubber**
# Objective
- Adding a new prop to make InlineNotification have it's first line of text underneath the notification heading

# Motivation and Context
- Closes this PR https://github.com/cultureamp/kaizen-design-system/issues/1843

# Screenshots (if appropriate)
### Non-dismissable
Screen Shot 2021-08-24 at 10 55 44 am

### Dismissable
Screen Shot 2021-08-24 at 10 55 51 am

# Checklist
- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

**Comment by dougmacknz** [comment]
🤔 I think you may have misunderstood the issue. My understanding is that @Hannah-de would like this outcome (rather than wanting the width of the container to be different):
image

**Comment by lloydstubber** [comment]
🤦 I was looking at the red line as a "we want it this wide". Will update.

**Comment by lloydstubber** [comment]
@dougmacknz updated

File: packages/component-library/stories/InlineNotification.stories.tsx [hunk 1]
```diff
@@ -136,6 +147,19 @@ export const DismissibleMultiline = () => (
...
+    type="negative"
+    title="Negative"
+    automationId="notification1"
+    forceMultiline
+  >
+    {forceMultilineText}
+  </InlineNotification>
+)
+
+DismissibleMultiline.storyName = "Dismissible, Multiline"
```
Comment (dougmacknz) [inline packages/component-library/stories/InlineNotification.stories.tsx:161]: This line isn't doing anything currently. It needs to be updated with the correct story var and I think it's worth being explicit about calling this '_Forced_ Multiline'

File: packages/component-library/stories/InlineNotification.stories.tsx [hunk 2]
```diff
@@ -215,6 +239,20 @@ export const PersistentMultiline = () => (
...
+    title="Negative"
+    persistent
+    automationId="notification1"
+    forceMultiline
+  >
+    {forceMultilineText}
+  </InlineNotification>
+)
+
+PersistentMultiline.storyName = "Persistent, Multiline"
```
Comment (dougmacknz) [inline packages/component-library/stories/InlineNotification.stories.tsx:254]: Same with this one

**Review by dougmacknz — COMMENTED** [review]
Couple things, otherwise good to go

File: packages/component-library/components/Notification/components/GenericNotification.tsx [hunk 3]
```diff
@@ -130,6 +131,12 @@ class GenericNotification extends React.Component<Props, State> {
     )
   }
 
+  textContainerClassName(): string {
+    return classnames(styles.textContainer, styles[this.props.style], {
```
Comment (dougmacknz) [inline packages/component-library/components/Notification/components/GenericNotification.tsx:135]: Actually, I'm just looking at the failed snapshot tests and are we sure we want the `styles[this.props.style]` part? 

Side note: did snapshots actually catch a real issue for once?

Comment (lloydstubber) [reply packages/component-library/components/Notification/components/GenericNotification.tsx:135]: Nope we don't need it! Sorry, getting use to the snapshots still 😅

**Review by dougmacknz — APPROVED** [review]

**Comment by Hannah-de** [comment]
Thank you @lloydstubber @dougmacknz 💥

**Pull request merged**